### DISCUSSION TITLE
I need a schema to view the SQL tables

### DISCUSSION THREAD

**Discussion by pierre-pedrito** [ccbogel/QualCoder#896] [Q&A] [answered]
Hello,
Would it be possible to know if there is a way of viewing the different tables in Qualcoder to understand the connections between the tables (via the identifier variable)?
This would make it easier for me to perform joins.
Thank you

**Comment by ccbogel** [comment, marked as answer]
Hi. There is this page on the wiki, which covers most things:
https://github.com/ccbogel/QualCoder/wiki/15-Sqlite-data-structure

But to do a visual version, it would take me a little while to create the image.

**Reply by pierre-pedrito** [reply]
Thank you very much for your reply. I couldn't find this section, because I haven't deployed the wiki menu completely (and I didn't see that I could deploy it).
